### PR TITLE
✨ add search_after param

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: modules/mapping-utils/src/createConnectionResolvers.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
+// @flow
 import { get } from 'lodash';
 
 import resolveAggregations from './resolveAggregations';
 import resolveHits from './resolveHits';
 
-type TcreateConnectionResolversArgs = {
+type TCreateConnectionResolversArgs = {
```
Comment (alex-wilmer) [inline modules/mapping-utils/src/createConnectionResolvers.js]: not that this is the most important line of code in the entire application, but I kind of prefer it the other way. it's like:

```
someFn
```

```
TsomeFn // type information
```

capitalizing the `s` seems weird to me

File: modules/mapping-utils/src/resolveHits.js [hunk 2]
```diff
@@ -83,14 +87,18 @@ export default type => async (
...
+
+    return {
+      searchAfter: JSON.stringify(x.sort || []),
+      node: { id: x._id, ...source, ...nested_nodes },
+    };
   });
 
   return {
-    hits: nodes,
+    edges,
```
Comment (alex-wilmer) [inline modules/mapping-utils/src/resolveHits.js]: is this not a fairly major / breaking change?

Comment (Jephuff) [reply modules/mapping-utils/src/resolveHits.js]: nope! that change along with https://github.com/overture-stack/arranger/pull/269/files#diff-d51be1b4aaffd8cba6ee072e7ff32159L53
makes the graphql response the same.

File: modules/mapping-utils/src/createConnectionResolvers.js (hunk 1 above)
Comment (Jephuff) [reply modules/mapping-utils/src/createConnectionResolvers.js]: hmmm, yeah I suppose. I can revert :) just feels weird to not have camel case haha

File: modules/mapping-utils/src/resolveHits.js (hunk 2 above)
Comment (alex-wilmer) [reply modules/mapping-utils/src/resolveHits.js]: ohh cool 👍

File: modules/mapping-utils/src/createConnectionResolvers.js (hunk 1 above)
Comment (Jephuff) [reply modules/mapping-utils/src/createConnectionResolvers.js]: reverted :)

**Pull request merged**